Gemfile

```diff
@@ -1,3 +1,5 @@
-source "http://www.rubygems.org"
+source "https://www.rubygems.org"
+
+gem 'mina', git: 'https://github.com/devvmh-forks/mina.git', branch: 'master'
```

Comment:
this can be removed once https://github.com/mina-deploy/mina/pull/415 is merged.